anycloud/docs/how-to/aws-postgres.md

```diff
@@ -0,0 +1,114 @@
+# Persist data to PostgreSQL in AWS
+
+The guide covers how to deploy a Node.js REST HTTP API to AWS using AnyCloud that also persists data to a PostgreSQL instance in AWS RDS. We will use the ORM [Prisma](https://www.prisma.io) and start from their [rest-express](https://github.com/prisma/prisma-examples/tree/latest/typescript/rest-express) example.
+
+All the code from this guide can be found in this [template repository](https://github.com/alantech/anycloud-node-aws-psql) which you can use to [create a new repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) for your AnyCloud project.
+
+## Create a PostgreSQL instance on AWS RDS
+
+Follow this [AWS guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html#CHAP_GettingStarted.Creating.PostgreSQL) to create a PostgreSQL on AWS RDS.
+
+## Enable programmatic AWS access to VMs for AnyCloud
+
+1) Create a new IAM user in your AWS account using their console/UI as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console).
+
+2) Create a new access key under that IAM user using their console/UI as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey).
+
+3) Enable programmatic access for that IAM user, and attach the built-in [`AmazonEC2FullAccess`](https://console.aws.amazon.com/iam/home#/policies/arn%3Aaws%3Aiam%3A%3Aaws%3Apolicy%2FAmazonEC2FullAccess)policy to it as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html#add-policies-console).
+
+4) Take the `accessKeyId` and `secretAccessKey` from step 2 and create an AWS `Credentials` in AnyCloud. You will need to pick a name or alias for the `Credentials`. The initial value will be `aws`. In this example, we will call it `mystartup-aws`.
+
+```bash
+$ anycloud credentials new
+? Pick cloud provider for the new credentials ›
+❯ AWS
+  GCP
+  Azure
+Name for new Credentials: mystartup-aws
+AWS Access Key ID: ******************
+AWS Secret Access Key: ******************
+Successfully created "mystartup-aws" Credentials
+```
+
+## Deploy the REST API using AnyCloud and link it to the PostgreSQL instance
+
+1) First download Prisma's example and install dependencies
+
+```
+curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/typescript/rest-express
```

Comment:
I've never seen anything like this before. Also, these instructions are missing the `git init` necessary for `anycloud` since it won't deploy outside of a git repo, so you could alternatively provide instructions like this to get the `rest-express` directory as a new repo: https://ao.gl/how-to-split-a-subdirectory-to-a-new-git-repository-and-keep-the-history/

```bash
git clone https://github.com/prisma/prisma-examples
cd prisma-examples
git filter-branch --prune-empty --subdirectory-filter typescript/rest-express
git remote remove origin
```

Reply:
Hmm not sure the user wants to preserve the git history of all the repository. I will add the `git init` to the current instructions though